test/api-callbacks.js

```diff
@@ -39,18 +39,12 @@ var auctionExample = require('./fixture/auctionexample1');
 
 describe('Api Callbacks - Tests', function() {
 
-  function clearEvents() {
-    Event.removeAllListeners();
-    Event.observeImmediate_ = null;
-    Event.observeImmediate_ = {};
-  }
-
   beforeEach(function() {
-    clearEvents();
+    Event.removeAllListeners();
   });
 
   afterEach(function() {
```

Comment:
Wrt the above, if `Event.removeAllListeners()` is not called `beforeEach`, the _api-callbacks.js_ test `AUCTION_COMPLETE` listener is invoked prematurely; there are no `bidProviderDoneCalled` keys defined an the test `done()` is called w/o errors.

Will uncomment [these lines in api-callbacks.js](https://github.com/pubfood/pubfood/blob/master/test/api-callbacks.js#L60-L62) to cover this for false positives.